crates/codec/Cargo.toml

```
@@ -0,0 +1,10 @@
+[package]
+name = "strata-codec"
+version = "0.1.0"
+edition = "2024"
```

```suggestion
edition = "2024"
readme = "README.md"
```

Since there's very relevant information in the `README.md`